Sources/SwiftInspectorVisitors/Tests/GenericRequirementVisitorSpec.swift

```diff
@@ -83,10 +82,9 @@ final class GenericRequirementVisitorSpec: QuickSpec {
             self.sut,
             overContent: content)
 
-          expect(self.sut.genericRequirements.first) == GenericRequirement(
```

Comment:
Why is `GenericRequirement` no longer equatable?

Reply:
it is! But I'm utilizing feedback I got from @fdiaz in another PR to only check the fields we care about, so we don't need to update each of these tests if we ever add a new property to `GenericRequirement`.

Reply:
Yeah, I've found that it's pretty annoying to have to modify every single test that does this when you add a new property so I've used 2 different patterns:

1) What Dan has here
2) Create a thing factory method in tests with defaults values `GenericRequirement.fake(leftType: String = "", rightType: String ="", ...` so when we add a new property we can just add it to this method instead with a default value and every test continues working